build-support-windows/BUILD-IMAGES.md

```diff
@@ -0,0 +1,119 @@
+# Dockerfiles for Windows Integration Tests

```

Comment:
can the move this directory as `/windows` underneath `./build-support` as a way to not expand the top level and embed this in the related topic of that directory for supporting build regardless of run time.